madgraph/fks/fks_helas_objects.py

```diff
@@ -830,6 +861,12 @@ def __eq__(self, other):
         othertag = helas_objects.IdentifyMETag.\
                         create_tag(other.born_me.get('base_amplitude'))
 
+        # MZ temporary fix for sudakov; since they are not taken into account
+        # in the combination, return False for the moment
+        if self.ewsudakov:
+            logger.warning('With --ewsudakov, matrix elements will not be combined')
+            return False
```

Comment:
So I guess this will not be temporary?
Maybe edit the comment?

Reply:
fixed